samples/event_manager_proxy/README.rst

```diff
@@ -74,6 +74,15 @@ For example, you can do this from the command line using west by completing the
       cd build/event_manager_proxy_remote-prefix/src/event_manager_proxy_remote-build/
       west flash
 
+Selecting ICMSG backend
+=======================
+
+The sample is provided with ICMSG backend configuration. To use it, specify the ``-DCONF_FILE=prj_icmsg.conf`` parameter along with the build command:
+
+  .. code-block:: console
+
+     west build -p -b nrf5340dk_nrf5340_cpuapp -- -DCONF_FILE=prj_icmsg.conf
+
```

Comment:
The whole steps should be moved after step 63 as we are building the sample at that step. The section heading should be removed.
```suggestion
 To provide the ICMSG backend configuration, specify the ``-DCONF_FILE=prj_icmsg.conf`` parameter along with the build command:

  .. code-bloc:: console

     west build -p -b nrf5340dk_nrf5340_cpuapp -- -DCONF_FILE=prj_icmsg.conf

```